packages/preview/strath-maestrum/0.0.1/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+entrypoint = "src/template.typ"
+name = "strath-maestrum"
+version = "0.0.1"
+authors = ["Barnaby Seaman <@barnaby2002>"]
+repository = "https://github.com/barnaby2002/strath-mae-interimst"
+description = "A template that replicates the Strathclyde MAE Interim Report"
+keywords = ["Strathclyde", "MAE", "Interim"]
+license = "MIT"
+categories = ["office"]
+exclude = ["template"]
```

Comment:
Please do not exclude your template files, they are required for your package to work properly.